CHANGELOG.md

```diff
@@ -27,4 +27,5 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 * Clean up parsing for query [#1824](https://github.com/opensearch-project/k-NN/pull/1824)
 * Refactor engine package structure [#1913](https://github.com/opensearch-project/k-NN/pull/1913)
 * Refactor method structure and definitions [#1920](https://github.com/opensearch-project/k-NN/pull/1920)
-* Generalize lib interface to return context objects [#1925](https://github.com/opensearch-project/k-NN/pull/1925)
\ No newline at end of file
+* Generalize lib interface to return context objects [#1925](https://github.com/opensearch-project/k-NN/pull/1925)
+* Moving query execution to shard level [#1877](https://github.com/opensearch-project/k-NN/pull/1877)
```

Comment:
That is not what we are doing. I would recommend changing this to 

Move `k` search k-NN query to re-write phase of vector search query for Native Engines.

Reason:
1. we are not making change in the radial search.
2. We are doing this for native engines.